<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppSettingsWindow</class>
 <widget class="QWidget" name="AppSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>586</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">appSetting
{
background-color: rgb(29, 31, 32);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="mainUIStyleFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="leftUIStyleFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>150</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="labelSeeUI">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>15</y>
           <width>58</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Вигляд&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="rightUIStyleFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>520</width>
          <height>150</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QRadioButton" name="selectSystemUIRadioButton">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>120</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Системна</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="selectWhiteUIRadioButton">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>120</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Світла</string>
         </property>
        </widget>
        <widget class="QLabel" name="imageWhiteUI">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>140</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid rgb(231,224,223);
border-radius: 8px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="recources.qrc">:/img/settingIcon/settingWhiteUIIco.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="imageBlackUI">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>30</y>
           <width>140</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid rgb(41,45,48); 
border-radius: 8px;

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="recources.qrc">:/img/settingIcon/settingBlackUIIco.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="selectBlackUIRadioButton">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>120</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Темна</string>
         </property>
        </widget>
        <widget class="QLabel" name="imageSystemUI">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>30</y>
           <width>140</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid rgb(41,45,48); 
border-radius: 8px;

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="recources.qrc">:/img/settingIcon/settingSystemUIICo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="selectSystemUIButton">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>10</y>
           <width>140</width>
           <height>131</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="selectBlackUIButton">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>10</y>
           <width>140</width>
           <height>131</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="selectWhiteUIButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>140</width>
           <height>131</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>selectSystemUIRadioButton</zorder>
        <zorder>selectWhiteUIRadioButton</zorder>
        <zorder>imageWhiteUI</zorder>
        <zorder>imageBlackUI</zorder>
        <zorder>imageSystemUI</zorder>
        <zorder>selectBlackUIRadioButton</zorder>
        <zorder>selectSystemUIButton</zorder>
        <zorder>selectBlackUIButton</zorder>
        <zorder>selectWhiteUIButton</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="connectSetting">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="leftConnectFrame">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>380</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>380</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="dataBaseText">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>15</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;База даних&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="accountText">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="rightConnectFrame">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>380</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>380</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="dbURL">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>40</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;URL бази даних&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="dbAPI">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>70</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;API ключ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="dbURLLineEdit">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>40</y>
           <width>341</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="dbAPILineEdit">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>70</y>
           <width>341</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="saveDBSettings">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>160</y>
           <width>100</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Зберегти</string>
         </property>
        </widget>
        <widget class="QLabel" name="succSaveSettings">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>160</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Налаштування збережені!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>160</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
